conf/messages

```diff
@@ -168,7 +168,8 @@ estimatedInterest.currentAmount = Current amount (estimate)
 estimatedInterest.amountReceived = Amount received
 estimatedInterest.leftToPay = Left to pay
 estimatedInterest.estimates = Estimates
-estimatedInterest.estimatesExplanation = Penalties and interest will show as estimates if HMRC has not been given enough information to calculate the final amounts.
+estimatedInterest.estimatesExplanation = Penalties and interest will show as estimates until you pay the charge they relate to.
+estimatedInterest.estimatesDescriptionAgent = Penalties and interest will show as estimates until your client pays the charge they relate to.
```

Comment:
Can you create an entry in the same place for this in the messages.cy file.

Reply:
Added entry in messages cy